mvrx/src/main/kotlin/com/airbnb/mvrx/DeliveryMode.kt

```diff
@@ -21,16 +21,16 @@ sealed class DeliveryMode {
  *
  * Likewise, when a [MavericksView] resubscribes after a configuration change the most recent update will always be emitted.
  */
-object RedeliverOnStart : DeliveryMode()
+object RedeliverOnStart : DeliveryMode(RedeliverOnStart::javaClass.name)
 
 /**
  * The subscription will receive the most recent state update when transitioning from locked to unlocked states (stopped -> started),
- * only if the state has changed while locked.
+ * only if the state has changed while locked. This will include the initial state as a state update.
  *
  * Likewise, when a [MavericksView] resubscribes after a configuration change the most recent update will only be emitted
  * if the state has changed while locked.
  *
  * @param subscriptionId A uniqueIdentifier for this subscription. It is an error for two unique only subscriptions to
  * have the same id.
  */
-class UniqueOnly(val subscriptionId: String) : DeliveryMode()
+class UniqueOnly(override val subscriptionId: String) : DeliveryMode(subscriptionId)
```

Comment:
it's redundant to pass the `subscriptionId` to the super class as a parameter, as well as override it here. Either don't make it open and pass it in the constructor, or take it out of the super constructor and put it in the body as an open property and override it here